jsplot/src/main/java/tech/tablesaw/plotly/components/Page.java

```diff
@@ -17,11 +17,18 @@
     private final Figure figure;
     private final String divName;
 
+    private String plotlyJsLocation = null;
+
     private Page(PageBuilder builder) {
         this.figure = builder.figure;
         this.divName = builder.divName;
+        this.plotlyJsLocation = builder.plotlyJsLocation;
     }
 
+    public void setPlotlyJsLocation(String plotlyJsLocation) {
```

Comment:
Could we remove this since it's unused and make `plotlyJsLocation` final?

Reply:
Of course, with a builder it is nonsense.